src/Console/ModelsCommand.php

```diff
@@ -17,8 +17,6 @@
 use Barryvdh\Reflection\DocBlock\Serializer as DocBlockSerializer;
 use Barryvdh\Reflection\DocBlock\Tag;
 use Composer\Autoload\ClassMapGenerator;
-use Doctrine\DBAL\Exception as DBALException;
-use Doctrine\DBAL\Types\Type;
```

Comment:
These lines should not be removed 🤔

Reply:
Sorry about that. I put the missing lines back.